Angular/GridApp/src/app/relation.service.ts

```diff
@@ -0,0 +1,26 @@
+import {Injectable} from '@angular/core';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {Relation} from './relation';
+    
+@Injectable()
+export class RelationService{
+    
+    private url = "https://localhost:44358/api/relations";
```

Comment:
https://localhost:44358 has to be moved to a separate file as a ServerBasePath constant so that it can be reused in the whole solution